src/main/java/com/tma/teamhr/service/impl/TeamServiceImpl.java

```diff
@@ -37,4 +38,17 @@ public TeamResponseDTO getById(int id) {
             throw new NullPointerException(message.NOTEXIST_ID);
         return new TeamResponseDTO(optionalTeam.get());
     }
+
+    @Override
+    public Boolean delete(int id) {
```

Comment:
change `Boolean` to `void` because there are just 2 cases: success or throw exception